### ISSUE TITLE
 Definition for rule 'jsdoc/newline-after-description'

### ISSUE THREAD

**Issue by kle-pra** [gajus/eslint-plugin-jsdoc#1031]
With  "eslint-plugin-jsdoc": "43.0.6" there seems to be an error with this rule and linting:

Example:
```
/home/user/project/frontend/src/polyfills.ts
  1:1  error  Definition for rule 'jsdoc/newline-after-description' was not found  jsdoc/newline-after-description
```

To pass linting I had to turn off  the rule off in eslintrc.json: `"jsdoc/newline-after-description": "off",
`
## Environment
    
 npm: '9.5.1',
 node: '18.16.0'

and:

"eslint": "^8.38.0",
"eslint-plugin-jsdoc": "43.0.6"

**Comment by brettz9** [comment]
The rule was removed. Are you relying on a config which uses it?

**Comment by brettz9** [comment]
I.e., what exactly does your .eslintrc or eslint.config.js file look like?

**Comment by kle-pra** [comment]
I see, I missed that.
Yes, config is extending from angular's  but it seems they are aware already. 
https://github.com/angular-eslint/angular-eslint/issues/1353

Thank you.

**Comment by HitkoDev** [comment]
For anyone else looking for a solution to this, see https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v42.0.0

**Comment by danvk** [comment]
If it helps anyone else… I used to have `{'jsdoc/newline-after-description': 'error'}` in my `.eslintrc`. After upgrading I have:

```js
{
  'jsdoc/tag-lines': [
    'error',
    'any',
    {
      startLines: 1,
    },
  ],
}
```

and I believe these are equivalent? [The docs](https://github.com/gajus/eslint-plugin-jsdoc/blob/3a5dd7d9f67fc01ec848758443c04da29d45327b/docs/rules/tag-lines.md#readme) for the `tag-lines` rule are quite hard to follow since there are no tags to speak of in the context of `newline-after-description`.
